cmd/atlas/internal/migrate/ent/entc.go

```diff
@@ -26,6 +26,9 @@ func main() {
 			gen.FeatureExecQuery,
 			gen.FeatureSchemaConfig,
 		},
+		Templates: []*gen.Template{
+			gen.MustParse(gen.NewTemplate("").ParseFiles("convert.tmpl")),
+		},
 	})
```

Comment:
Maybe pass it as an [`entc.Option`](https://github.com/ent/ent/blob/master/entc/entc.go#L159)? It will look cleaner:

```suggestion
	}, entc.TemplateFiles("convert.tmpl"))
```